pkg/storkctl/clusterpair.go

```diff
@@ -598,7 +606,7 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 	createClusterPairCommand.Flags().StringVarP(&destToken, "dest-token", "", "", "(Optional)Destination cluster token for cluster pairing")
 	createClusterPairCommand.Flags().StringVarP(&projectMappingsStr, "project-mappings", "", "", projectMappingHelpString)
 	createClusterPairCommand.Flags().StringVarP(&mode, "mode", "", "async-dr", "Mode of DR. [async-dr, sync-dr]")
-
+	createClusterPairCommand.Flags().BoolVarP(&uniDirectional, "uni-directional", "u", false, "Required to create Clusterpair from source -> dest only")
```

Comment:
Instead of "Required to", I think we can mention (Optional) To

Reply:
Done